RELEASE_NOTES.rst

```diff
@@ -21,7 +21,53 @@
 
 .. towncrier release notes start
 
-Airflow 2.6.0 (2023-04-28)
+Airflow 2.6.1 (2023-05-12)
+--------------------------
+
+Significant Changes
+^^^^^^^^^^^^^^^^^^^
+No significant changes.
+
+Bug Fixes
+^^^^^^^^^
+- Move interleave_timestamp_parser config to section logging (#31102)
+- fix MappedTaskGroup import in taskinstance (#31100)
+- Format DagBag.dagbag_report() Output (#31095)
+- Fix template error when iterating None value and fix params documentation (#31078)
+- fix EXTRAS_DEPENDENCIES of apache-hive (#31068)
+- Fix issue in pre-importing modules in zipfile (#31061)
+- Move TaskInstanceKey to a separate file (#31033)
+- Fix unable to remove DagRun and TaskInstance with note (#30987)
+- Fix ``airflow providers get`` command output (#30978)
+- Fix Pool schema OpenAPI spec (#30973)
+- Update incorrect config values (#30969)
+- Add support for dynamic tasks with template fields that contain ``pandas.DataFrame`` (#30943)
+- Use the Task Group explicitly passed to 'partial' if any (#30933)
+- Fix ``order_by`` request in list DAG rest api (#30926)
+- Include node height/width in center-on-task logic (#30924)
+- Remove print from dag trigger command (#30921)
+- Improve task group UI in new graph (#30918)
+- Fix mapped states case (#30916)
+- Fix problem with displaying graph (#30765)
+- Fix backfill KeyError when try_number out of sync (#30653)
+- Re-enable clear on TaskInstanceModelView for role User (#30415)
+- Fix DAG.access_control can't sync when clean access_control (#30340)
```

Comment:
I have a concern with this change. I think it fixes one issue (empty access_control isn't respected), but creates another (it wipes out any rights not set by access_control initially).

Reply:
Then I will drop it